File: Python/tests/test_bart_par4.py
import pytest

from hbayesdm.models import bart_par4


def test_bart_par4():
    _ = bart_par4(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_bart_par4.R
context("Test bart_par4")
library(hBayesDM)

test_that("Test bart_par4", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(bart_par4(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
